thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/DetectionException.java

```diff
@@ -0,0 +1,22 @@
+package org.apache.pinot.thirdeye.detection;
+
+/**
+ * The exception to be thrown in detection pipelines.
+ */
+public class DetectionException extends Exception {
```

Comment:
We already have DetectorException in spi. Please use that.

Reply:
We can separate the exceptions thrown in the Detector implementation and the Detection pipeline wrappers so that we can know where the exception is thrown and be more specific in error handling.

Reply:
Sounds good. Let's keep it.

Reply:
So I renamed it to DetectionPipelineException. what do you think?